src/main/java/reposense/parser/CsvParser.java

```diff
@@ -96,18 +96,17 @@ protected String getValueInElement(final String[] elements, int position) {
 
     /**
      * Gets the value of {@code position} in {@code elements}.
-     * Returns the value of {@code position} in an array, delimited by {@code AUTHOR_ALIAS_AND_GLOB_SEPARATOR}
+     * Returns the value of {@code position} in a {@code List}, delimited by {@code COLUMN_VALUES_SEPARATOR}
      * if it is in {@code element} and not empty.
-     * Otherwise returns an empty array.
+     * Otherwise returns an empty {@code List}.
      */
     protected List<String> getManyValueInElement(final String[] elements, int position) {
         if (!containsValueAtPosition(elements, position)) {
             return Collections.emptyList();
         }
 
         String manyValue = getValueInElement(elements, position);
-        // Wrap with new ArrayList<> to support all list functionalities.
-        return new ArrayList<>(Arrays.asList(manyValue.split(AUTHOR_ALIAS_AND_GLOB_SEPARATOR)));
+        return Arrays.stream(manyValue.split(COLUMN_VALUES_SEPARATOR)).map(String::trim).collect(Collectors.toList());
```

Comment:
is there a need to change this? `Stream` has slower performance compare to the previous.

Reply:
I realize I should trim spaces. `i.e. java   ;   adoc`.